tests/unit/package_managers/yarn_classic/test_project.py

```diff
@@ -230,6 +230,16 @@ def test_pnp_installs_false(
         config_file_name,
         config_file_content,
     )
+    # A yarn v1 project needs a valid yarn lock to be accepted.
+    _prepare_config_file(
+        rooted_tmp_path,
+        # I will deal with
+        #   incompatible type "type[YarnLock]"; expected "Union[PackageJson, YarnLock]"
+        # later, will ignore mypy for now:
```

Comment:
I would suggest to drop personalized comments / notes and create issues instead for example.